src/unstable-temp/DownloadImageModal/DownloadImageModal.tsx

```diff
@@ -101,7 +101,7 @@ export interface UnstableTempDownloadImageModalProps {
 	getDownloadSize?: (
 		deviceType: DeviceType,
 		rawVersion: string | null,
-	) => Promise<string> | undefined;
+	) => Promise<string>;
```

Comment:
Based on how the code was, returning `undefined` would have made the component throw anyway, so we are now just making the types stricter.